server/src/main/java/org/elasticsearch/cluster/routing/allocation/Explanations.java

```diff
@@ -29,10 +29,10 @@ public static final class Allocation {
             will allocate this shard when a node containing a good copy of its data joins the cluster. If no such node is available, \
             restore this index from a recent snapshot.""";
 
-        public static final String NO_COPIES = """
+        public static final String NO_COPIES = String.format("""
```

Comment:
This API is forbidden - you forgot to run `./gradlew precommit` before opening this.

Reply:
💬 internal